src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -96,6 +101,8 @@ public string InstanceId
 
         public ScriptHostConfiguration ScriptConfig { get; private set; }
 
+        public IDictionary<string, IAsyncConverter<HttpRequestMessage, HttpResponseMessage>> CustomHttpHandlers { get { return _customHttpHandlers; } }
```

Comment:
what's that mean?

---
In reply to: [108762541](https://github.com/Azure/azure-webjobs-sdk-script/pull/1349#discussion_r108762541) [](ancestors = 108762541)